src/main/scala/com/amazon/deequ/analyzers/KLLSketch.scala

```diff
@@ -77,18 +77,27 @@ object KLLState{
  * The KLL Sketch analyzer.
  * @param column the column to run the analyzer
  * @param where constraint expression on the column
- * @param sketchSize Sketch size of KLL Sketch
- * @param shrinkingFactor Shrinking factor of KLL Sketch
- * @param numberOfBuckets number of buckets to run statistics on the samples
+ * @param kllParameters parameters of KLL Sketch
  */
 case class KLLSketch(
     column: String,
     where: Option[String] = None,
-    sketchSize: Int = KLLSketch.DEFAULT_SKETCH_SIZE,
-    shrinkingFactor: Double = KLLSketch.DEFAULT_SHRINKING_FACTOR,
-    numberOfBuckets: Int = KLLSketch.MAXIMUM_ALLOWED_DETAIL_BINS)
+    kllParameters: Option[Seq[Double]] = None)
```

Comment:
I would prefer a case class to a Seq here, because the semantics would be clearer.

Reply:
Great idea! Done.